core-web/src/main/java/org/silverpeas/core/web/treemenu/process/TreeBuilder.java

```diff
@@ -117,10 +119,13 @@ private static MenuItem buildOtherLevel(TreeFilter filter, MenuItem father, Stri
       // gets the sub space
       List<SpaceInstLight> subspaces =
           controller.getSubSpacesContainingComponent(father.getKey(), userId, "kmelia");
+      if (TreeHandler.useOrder)
+        subspaces.sort(Comparator.comparing(SpaceInstLight::getOrderNum));
+
```

Comment:
A Java rule: always the body of an if or else-statements is put in brackets, even if there is only a single statement of code

Reply:
Done.